src/App.tsx

```diff
@@ -8,37 +8,205 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { getAllUsers } from './api/users';
+import { User } from './types/User';
+import { getUserPosts } from './api/posts';
+import { Post } from './types/Post';
+import { Comment, CommentData } from './types/Comment';
+import { getPostComments, addNewComment, deleteComment } from './api/comments';
+
+const tempUser = {
+  id: 0,
+  name: '',
+  email: '',
+  phone: '',
+};
+
+const tempPost = {
+  id: 0,
+  userId: 0,
+  title: '',
+  body: '',
+};
+
+type SendDataType = {
```

Comment:
Consider renaming `SendDataType` to a more descriptive name, such as `CommentSendData`.